docs/headless-api.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Rojo Plugin APIs
+sidebar_position: 7
+---
+
+The Rojo Studio plugin exposes a headless API into `game.Rojo`, which is very valuable to the Rojo ecosystem. This allows you to make companion plugins that interact with Rojo!
+
+## Accessing the API
+
+Plugins and command bar all share the same module cache, which allows us to expose the API through a ModuleScript located at `game.Rojo`. Using the name `"Rojo"` is easy to remember and follows branding. Once you've required Rojo, the API will be located at `Rojo.API`. For example:
```

Comment:
This phrasing still reads a little oddly imo - maybe we should simply say "We chose the name `Rojo` because it is easy to remember," or alternatively, strike this sentence entirely? I don't think it's particularly necessary for the sentiment we're trying to get across here